amundsen_application/static/js/components/common/ShimmeringTagListLoader/styles.scss

```diff
@@ -18,10 +18,6 @@ $shimmer-loader-tag-min-width: 50;
 
 @each $item in $shimmer-loader-items {
   .shimmer-tag-loader-item--#{$item} {
-    width: (
-        random($shimmer-loader-tag-max-width - $shimmer-loader-tag-min-width) +
-          $shimmer-loader-tag-min-width
-      ) +
-      px;
+    width: $shimmer-loader-tag-width + px;
   }
 }
```

Comment:
Actually, you can remove this whole thing and use the $shimmer-loader-tag-width on line 14!